### PR TITLE
add alfred-link note

### PR DESCRIPTION
**Pull request by SamVerschueren**
Want to resolve https://github.com/SamVerschueren/alfred-link/issues/2 first and add an extra paragraph here as well referencing the moot properties in `info.plist`. So feedback about the structure, explanation, etc are more then welcome. But do not merge it yet :).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -68,6 +68,42 @@ alfy.fetch('jsonplaceholder.typicode.com/posts').then(data => {
 Some example usage in the wild: [`alfred-npms`](https://github.com/sindresorhus/alfred-npms), [`alfred-emoj`](https://github.com/sindresorhus/alfred-emoj), [`alfred-ng2`](https://github.com/SamVerschueren/alfred-ng2).
 
 
+## Publish to npm
+
+Instead of publishing your packages to [Packal](http://www.packal.org/), you can also publish them to [npm](https://npmjs.org). This way, your packages
+are only one simple `npm install` command away.
```
Comment (sindresorhus) [inline readme.md]: Don't hardwrap

File: readme.md [hunk 2]
```diff
@@ -68,6 +68,42 @@ alfy.fetch('jsonplaceholder.typicode.com/posts').then(data => {
 Some example usage in the wild: [`alfred-npms`](https://github.com/sindresorhus/alfred-npms), [`alfred-emoj`](https://github.com/sindresorhus/alfred-emoj), [`alfred-ng2`](https://github.com/SamVerschueren/alfred-ng2).
 
 
+## Publish to npm
+
+Instead of publishing your packages to [Packal](http://www.packal.org/), you can also publish them to [npm](https://npmjs.org). This way, your packages
```
Comment (sindresorhus) [inline readme.md]: Remove the `/` in the Packal URL

File: readme.md [hunk 3]
```diff
@@ -68,6 +68,42 @@ alfy.fetch('jsonplaceholder.typicode.com/posts').then(data => {
...
+
+```json
+{
+  "name": "alfred-unicorn",
+  "version": "1.0.0",
+  "description": "My awesome unicorn workflow",
+  "author": {
+    "name": "Sindre Sorhus",
+	"email": "sindresorhus@gmail.com",
+	"url": "sindresorhus.com"
```
Comment (sindresorhus) [inline readme.md]: Wrong indent

**Comment by SamVerschueren** [comment]
Tests seems quite flaky. Might be caused by the `m.cache.clear()` we use in in 2 separate test files. Any ideas except from merging all the test files in one single file?

**Comment by SamVerschueren** [comment]
Added the paragraph regarding the moot properties in `info.plist`.

File: readme.md [hunk 4]
```diff
@@ -68,6 +68,43 @@ alfy.fetch('jsonplaceholder.typicode.com/posts').then(data => {
...
+  "dependencies": {
+    "alfy": "*",
+    "alfred-link": "*"
+  }
+}
+```
+
+> Tip: Prefix your workflow with `alfred-` to make them easy searchable through npm.
+
+You can remove [these](https://github.com/samverschueren/alfred-link#infoplist) properties from your `info.plist` file as they are being added automatically on install time.
```
Comment (sindresorhus) [inline readme.md]: `on` => `at`?

**Pull request merged**